src/components/Header/Header.css

```diff
@@ -1,56 +1,328 @@
 .header {
-    background-color: #242424;
-    border-radius: 14px;
-    position: relative;
-    padding: 20px 100px 140px 100px;
-    margin: 10px;
-    box-sizing: content-box;
+  display: grid;
+  grid-template-columns: 3fr;
+  background-color: #242424;
+  border-radius: 14px;
+  position: relative;
+  padding: 140px 100px;
+  margin: 10px;
+  box-sizing: content-box;
 }
 
 .header__main {
-    position: relative;
-    max-width: 330px;
-    margin: 0 auto;
-    display: flex;
-    flex-direction: column;
-    margin-top: 160px;
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  margin-top: 90px;
 }
 
 .header__logo {
-    position: absolute;
-    top: 0;
-    left: 37px;
+  width: 23px;
+  height: 28px;
+  position: absolute;
+  top: 0;
+  left: 37px;
 }
 
 .header__title {
-    color: #fff;
-    font-size: 34px;
-    font-style: normal;
-    font-weight: 700;
-    line-height: 43px;
-    letter-spacing: 0em;
-    text-align: center;
-    margin: 0;
-    padding: 0;
-    padding-top: 28px;
+  color: #fff;
+  font-size: 34px;
+  font-style: normal;
```

Comment:
Might be best to remove superfluous styles, like `font-style: normal` and `letter-spacing: 0em`.  Often the `text-align` property that you copy over from Figma is uneccessary as well, but that depends on the specific use cases.